src/plugins/intel_cpu/src/nodes/string_tensor_unpack.cpp

```diff
@@ -75,22 +74,13 @@ void StringTensorUnpack::executeImpl() {
         string_count);
 }
 
-namespace {
-struct StringTensorUnpackContext {
-    StringTensorUnpack &node;
-};
-}
-
 struct StringTensorUnpack::StringTensorUnpackExecute {
-    void operator()(StringTensorUnpackContext & ctx) {
-            ctx.node.executeImpl();
+    void operator()(StringTensorUnpack& node) {
+            node.executeImpl();
         }
 };
```

Comment:
Why can't we simply call `executeImpl()` inside the `execute` method? Moreover we can inline `executeImpl()` code inside the `execute` method as the `executeImpl()` is used once.

Reply:
Good point, done.